src/ResourceManager/RecoveryServices.Backup/CmdletParameterHelpMessages.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
+{
+    internal static class ParamHelpMsg
+    {
+        internal static class Container
+        {
+            internal static class Get
```

Comment:
I agree with moving the Vault help message to common. For other common properties across cmdlets, we can define const vars which can be reused by various properties. This will limit any randomization at the cmdlet layer - all the required properties will be at one place.